spec/helpers/regions_helper_spec.rb

```diff
@@ -19,7 +19,7 @@
 
     it 'renders without a wrapper tag if no tag provided' do
       content = render_region(:body)
-      content.should have_selector('h1', content: 'Fred Flinstone')
+      content.should have_selector('h1', text: 'Fred Flinstone')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.